src/components/CookieToast.js

```diff
@@ -194,11 +194,9 @@ export default function CookieToast() {
                           onClick={() => {
                             setShowToast(false)
                             Cookie.set('accept_cookies', true, { expires: 365 })
-                            {
-                              checkbox === true
+                            checkbox === true
```

Comment:
why not
`Cookie.set('accept_analytics', !!checkbox, { expires: 365 })` ?

Reply:
Done